README.md

```diff
@@ -1,23 +1,16 @@
-# gitops-working-group
+# GitOps Working Group
 
-## Announcing the GitOps Working Group
+The GitOps Working Group is a WG under the CNCF App Delivery SIG.
 
-On November 19, 2020, Amazon, Codefresh, GitHub, Microsoft, and Weaveworks are announcing the creation of the GitOps Working Group.
-This will be an open CNCF community project created inside the CNCF [gitops-working-group GitHub organization](https://github.com/gitops-working-group) as the initial venue for collaboration and open governance.
+The focus of the GitOps WG is to clearly define a vendor-neutral, principle-led meaning of GitOps. This will establish a foundation for interoperability between tools, conformance, and certification. Lasting programs, documents, and code are planned to live within the [OpenGitOps](https://github.com/open-gitops) project.
```

Comment:
Sorry, what does lasting programs and documents imply? :)

Reply:
@sbose78 any documents that the WG put together that are meant to live on. Examples:
- GitOps Principles
- Any conformance and certification material
- Programs such as certification, trainings, etc

Hope that helps clarify